cmd/generate-plugin-overview/go.sum

```diff
@@ -122,6 +122,7 @@ sigs.k8s.io/krew v0.2.2-0.20190724210953-8867a94acd6d h1:ftkzey0SeGithDvmwEsUnlD
 sigs.k8s.io/krew v0.2.2-0.20190724210953-8867a94acd6d/go.mod h1:wsheYhp5znrUApov8QcNjBCwXu85fIGjRJvjSkE8pI8=
 sigs.k8s.io/krew v0.3.2 h1:FKBD9xy9GtO5A8FeVk/cbgG6vzPbUg4SwUziZj/TGoU=
 sigs.k8s.io/krew v0.3.2/go.mod h1:G0cvdJtBb0gHBQZtCBumtllvhU6H+ggDr2oLznaLglc=
+sigs.k8s.io/krew v0.3.3 h1:cHv2oReJmFmR0CnT9Fs1c9JCAOC6OBpdq4xKEJis+2E=
```

Comment:
why do we have separate go.mod files ?

Reply:
This diff is intentional to discuss this issue otherwise, it shouldn't be a part of this PR.

Reply:
This is intentional.
The stuff under cmd/* are essentially like different repos that have separate set of dependencies and they rely on parts of krew.

Reply:
But they will move further altogether, right? We won't release them separately.

Reply:
Right now, it's only added for `generate-plugin-overview` cmd, I suggest dropping it.

Reply:
Yeah, they move together. We can make them part of the main repo, by removing go.mod.
@corneliusweig wdyt?

Reply:
BTW I think the main reason these cmd/* tools have their own go.mod is mostly so that we don't download _their_ dependencies while building krew.

Reply:
I see your point but no extra dependency so it is somehow premature optimization.

Reply:
I sent https://github.com/kubernetes-sigs/krew/pull/424 to remove `go.mod`.

Reply:
Yeah, make sense. It also reduces the maintenance overhead of keeping the child module in sync with the current krew version :+1: